src/PrestaShopBundle/Form/Admin/Sell/Order/CancelProductType.php

```diff
@@ -147,10 +148,11 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             ])
             ->add('save', SubmitType::class, [
                 'attr' => [
-                    'class' => 'cancel-product-element save btn btn-primary ml-3',
+                    'class' => 'partial-refund save btn btn-primary ml-3',
```

Comment:
Nope, wrong rebase or outdated code I think This calss is not used any more, the javascript relies on the `cancel-product-element` class for its toggling selectors

Reply:
done